toggle-dummy.js

```diff
@@ -1,8 +1,27 @@
-  $(function(){
-    var button =
-      $('<button type="button" onclick="toggleDummy()"> Show dummies</button>');
-    $("body").prepend(button);
-  });
-  function toggleDummy(){
-    $(".dummy").toggleClass("is-active")
+$(function(){
+  addButton()
+  if (dummyOn()){
+    toggleDummy();
   }
+});
+
+function addButton() {
+  var button = $('<button type="button" onclick="toggleDummy();toggleStorage()"> Show dummies</button>');
+  $("body").prepend(button);
+}
+
+function toggleDummy(){
+  $(".dummy").toggleClass("is-active");
+}
+
+function toggleStorage() {
+  if ( dummyOn() ) {
```

Comment:
@drapergeek Explain what you mean by inverting it. I don't know how to invert it besides checking what it is and then doing the opposite...